app/src/main/java/kr/co/nottodo/presentation/toplevel/recommendation/recommendationfragment/RecommendationAdapter.kt

```diff
@@ -0,0 +1,102 @@
+package kr.co.nottodo.presentation.toplevel.recommendation.recommendationfragment
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import kr.co.nottodo.R
+import kr.co.nottodo.databinding.ItemRecommendBinding
+import kr.co.nottodo.presentation.toplevel.recommendation.data.recommendationlistdata.RecommendationData
+import kr.co.nottodo.presentation.toplevel.recommendation.data.responsedto.ResponseRecommendationCategorySituationDto
+
+class RecommendationAdapter(
+    private val sampleList: List<ResponseRecommendationCategorySituationDto.CategorySituation>,
+) : RecyclerView.Adapter<RecommendationViewHolder>() {
+
+    private var selectedPosition = -1
+    private var lastItemSelectedPosition = -1
```

Comment:
```suggestion
    private var selectedPosition = RecyclerView.NO_POSITION
    private var lastItemSelectedPosition = RecyclerView.NO_POSITION
```

이미 이런 상황을 나타내는 값이 리사이클러뷰 자체에 내장되어 있어서 사용하면 좋을 것 같아!